org.contourterminal.Contour.desktop

```diff
@@ -0,0 +1,196 @@
+# TODO
+# - [ ] make Exec prefix configurable via @CMAKE_INSTALL_PREFIX@/bin
+# - [ ] include it as install target
+
+[Desktop Entry]
```

Comment:
there is a .desktop file upstream it should be used instead.

Reply:
I am upstream. It's a copy from the upstream repo with the problems as raised by the validation step fixed. If it passes here (including the AppStream XML file), i.e. if flathub CI won't complain anymore, I'll gladly move it over and create a new release out of it.